examples/scripts/README.rst

```diff
@@ -10,74 +10,74 @@ Computed Tomography
+      CT Reconstruction with CG and PCG
    `ct_astra_tv_admm.py <ct_astra_tv_admm.py>`_
-      Few-View CT (ADMM w/ Total Variation)
+      TV Regularized Few-View CT Reconstruction
    `ct_astra_weighted_tv_admm.py <ct_astra_weighted_tv_admm.py>`_
-      Low-Dose CT (ADMM w/ Total Variation)
+      TV Regularized Low-Dose CT Reconstruction
+   `ct_svmbir_tv_multi.py <ct_svmbir_tv_multi.py>`_
+      TV Regularized CT Reconstruction (Multiple Algorithms)
    `ct_svmbir_ppp_bm3d_admm_cg.py <ct_svmbir_ppp_bm3d_admm_cg.py>`_
-      CT Reconstruction (ADMM Plug-and-Play Priors w/ BM3D, SVMBIR+CG)
+      PPP (with BM3D) CT Reconstruction (ADMM with CG Subproblem Solver)
    `ct_svmbir_ppp_bm3d_admm_prox.py <ct_svmbir_ppp_bm3d_admm_prox.py>`_
-      CT Reconstruction (ADMM Plug-and-Play Priors w/ BM3D, SVMBIR+Prox)
+      PPP (with BM3D) CT Reconstruction (ADMM with SVMBIR Fast Prox)
    `ct_fan_svmbir_ppp_bm3d_admm_prox.py <ct_fan_svmbir_ppp_bm3d_admm_prox.py>`_
-      Fan-beam CT Reconstruction (ADMM Plug-and-Play Priors w/ BM3D, SVMBIR+Prox)
-   `ct_svmbir_tv_multi.py <ct_svmbir_tv_multi.py>`_
-      CT Reconstruction with TV Regularization
+      PPP (with BM3D) Fan-Beam CT Reconstruction
 
 
 Deconvolution
 ^^^^^^^^^^^^^
 
    `deconv_circ_tv_admm.py <deconv_circ_tv_admm.py>`_
-      Image Deconvolution (ADMM w/ Total Variation and Circulant Blur)
+      Circulant Blur Image Deconvolution with TV Regularization
+   `deconv_tv_admm.py <deconv_tv_admm.py>`_
+      Image Deconvolution with TV Regularization
+   `deconv_tv_admm_tune.py <deconv_tv_admm_tune.py>`_
+      Image Deconvolution Parameter Tuning
    `deconv_microscopy_tv_admm.py <deconv_microscopy_tv_admm.py>`_
       Deconvolution Microscopy (Single Channel)
    `deconv_microscopy_allchn_tv_admm.py <deconv_microscopy_allchn_tv_admm.py>`_
       Deconvolution Microscopy (All Channels)
    `deconv_ppp_bm3d_admm.py <deconv_ppp_bm3d_admm.py>`_
-      Image Deconvolution (ADMM Plug-and-Play Priors w/ BM3D)
+      PPP (with BM3D) Image Deconvolution (ADMM Solver)
    `deconv_ppp_bm3d_pgm.py <deconv_ppp_bm3d_pgm.py>`_
-      Image Deconvolution (PGM Plug-and-Play Priors w/ BM3D)
-   `deconv_ppp_bm4d_admm.py <deconv_ppp_bm4d_admm.py>`_
-      Image Deconvolution (ADMM Plug-and-Play Priors w/ BM4D)
+      PPP (with BM3D) Image Deconvolution (APGM Solver)
    `deconv_ppp_dncnn_admm.py <deconv_ppp_dncnn_admm.py>`_
-      Image Deconvolution (ADMM Plug-and-Play Priors w/ DnCNN)
-   `deconv_tv_admm.py <deconv_tv_admm.py>`_
-      Image Deconvolution (ADMM w/ Total Variation)
-   `deconv_tv_admm_tune.py <deconv_tv_admm_tune.py>`_
-      Image Deconvolution Parameter Tuning
+      PPP (with DnCNN) Image Deconvolution
+   `deconv_ppp_bm4d_admm.py <deconv_ppp_bm4d_admm.py>`_
+      PPP (with BM4D) Volume Deconvolution
 
 
 Sparse Coding
 ^^^^^^^^^^^^^
 
    `sparsecode_admm.py <sparsecode_admm.py>`_
-      Non-negative Basis Pursuit DeNoising (ADMM)
+      Non-Negative Basis Pursuit DeNoising (ADMM)
```

Comment:
Did we find a consensus on how we handle capitalization?
I find it odd to read `COde` or `DeNoising`
@Michael-T-McCann

Reply:
This is non-standard capitalization, but it's the one originally used to justify the BPDN acronym. I'm open to alternatives if anyone finds this version truly offensive.